src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/designforextension/InputDesignForExtension.java

```diff
@@ -0,0 +1,117 @@
+////////////////////////////////////////////////////////////////////////////////
+// Test case file for checkstyle.
+// Created: 2001
+////////////////////////////////////////////////////////////////////////////////
+package com.puppycrawl.tools.checkstyle.checks.design.designforextension;
+
+/**
+ * Test case for the "design for inheritance" check.
+ * @author Lars Kühne
+ **/
+public abstract class InputDesignForExtension
```

Comment:
@Kietzmann Please show us where.
https://github.com/checkstyle/checkstyle/search?utf8=%E2%9C%93&q=InputDesignForExtension&type=

It doesn't show up on a search.